eoscostarica/eden-spend-explorer
# feat(webapp): option to select all election years and last election

### Option to select all election years and last election

### What does this PR do?

- Resolve #125 

### Steps to test

1. run the project and open localhost:3000 in your browser 
2. Click on income on the sidebar
3. Click on one Election radio button
4. Now you can see all by default in year select and election select you can see all election

#### CheckList


![image](https://user-images.githubusercontent.com/56526123/192351391-938f811a-5256-488c-a56b-7b620170edd8.png)
